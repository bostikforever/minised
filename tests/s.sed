s/some text/other text/
