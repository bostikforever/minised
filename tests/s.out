This is other text, that should not stay some text.
